resource/io/src/lib.rs

```diff
@@ -0,0 +1,125 @@
+#![no_std]
+
+use codec::{Decode, Encode};
+use gstd::prelude::*;
+use scale_info::TypeInfo;
```

Comment:
```suggestion
use gstd::prelude::*;
```
These traits have already been imported by `gstd::prelude::*`.